Themes/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "cpptools-themes",
+  "displayName": "C/C++ Themes",
+  "description": "UI Themes for C/C++ extension.",
+  "version": "0.24.1",
```

Comment:
Update the version.

Reply:
Should this have its own version number?  These would be in their own vsix, and we probably don't want to have to update its version every time we update cpptools.

Reply:
Sorry, that's what I meant.  This would be version 1.0.0 probably.